src/coreclr/vm/clsload.cpp

```diff
@@ -3708,6 +3484,7 @@ VOID ClassLoader::AddAvailableClassDontHaveLock(Module *pModule,
     AddAvailableClassHaveLock(
         pModule,
         classdef,
+        NULL,
```

Comment:
Instead of `NULL`, you could declare `SArray<EEClassHashEntry_t *> classEntries();` above and it will have 0 length by construction. It avoids creating a `NULL` check semantic in this code path.

Reply:
Sure. This reduces the complexity of the code a bit.